docs/Other SDL Features/Setting the Built-In Navigation Destination/index.md

```diff
@@ -0,0 +1,155 @@
+# Setting the Built-In Navigation Destination
```

Comment:
Change this to `Setting the Navigation Destination`

Reply:
Fixed